components/arrow_ext/src/ipc.rs

```diff
@@ -48,13 +51,22 @@ pub struct RecordBatchesEncoder {
     compress_opts: CompressOptions,
 }
 
-#[derive(Debug, Clone, Copy, Default)]
+#[derive(Debug, Clone, Copy, Deserialize, Serialize)]
 pub struct CompressOptions {
     /// The minimum length of the payload to be compressed.
     pub compress_min_length: usize,
     pub method: CompressionMethod,
 }
 
+impl Default for CompressOptions {
+    fn default() -> Self {
+        Self {
+            compress_min_length: DEFAULT_COMPRESS_MIN_LENGTH,
+            method: CompressionMethod::None,
```

Comment:
Those value default to zstd before, why change to none?

Reply:
In my opinion, in most scenarios, network traffic is not the limiting factor for a system, so there is no need to default to using zstd compression.